spec/spec_params.rb

```diff
@@ -64,10 +64,12 @@ module OptimizelySpec
       'percentageIncluded' => 10_000,
       'variations' => [{
         'key' => 'control',
-        'id' => '111128'
+        'id' => '111128',
+        Optimizely::Helpers::Constants::FEATURE_ENABLED => true
```

Comment:
Don't use the same constant everywhere.  How will you verify that the property has been named as intended?
In project_config.rb hardcoded it as others have been hardcoded. 
In data file hard code it. 
In spec file, keep it as a constant in the same file